tests/behat/theme_boost_union_feelsettings_navigation.feature

```diff
@@ -16,6 +16,11 @@ Feature: Configuring the theme_boost_union plugin for the "Navigation" tab on th
       | user     | course | role           |
       | teacher1 | C1     | editingteacher |
       | student1 | C1     | student        |
+    And the following "activities" exist:
```

Comment:
Nice to have:
I would like to propose to move this section out of the background and into each of the "Setting: Activity navigation" scenarios. Doing this, you will have this section twice in this feature file. But you won't burn CPU time uselessly in all other scenarios in this feature file which do not need these activities at all.

Reply:
Good point. Done!